iocore/net/UnixNetAccept.cc

```diff
@@ -260,15 +260,6 @@ NetAccept::do_listen(bool non_blocking)
     }
   }
 
-  if (opt.f_callback_on_open && !action_->cancelled) {
```

Comment:
I don't understand this at all - this isn't calling back on an _accept_, but on "port open".